tests/unit/test_bigbang.py

```diff
@@ -195,9 +197,10 @@ def test_activity_summary(self):
 class TestArchive(unittest.TestCase):
     def test_mailman_chain(self):
         name = "bigbang-dev-test.txt"
+        dir = project_directory + "/tests/data"
```

Comment:
I think this is just `CONFIG.test_data_path`, which would be clearer and not rely on joining paths through string concatenation.

Reply:
good point